File: Modul 4/SOAL 1/PRAK301-2210817120003-IKHTIARA FAKHRUNISA.c
#include <stdio.h>

int main() {
    for(int i = 0; i <= 2; i++){
        int angka1, angka2;
        scanf("%d %d", &angka1, &angka2);

        if(angka1 < angka2) {
            printf("%d %d", angka1, angka2);
        }
        else {
            printf("%d %d", angka2, angka1);
        }}

        return 0;
    }

File: Modul 4/SOAL 1/PRAK301-2210817120003-IKHTIARA FAKHRUNISA.py
for x in range(3):
    angka1, angka2 = input("").split()
    angka1 = int(angka1)
    angka2 = int(angka2)

    if angka1 < angka2 :
        print(angka1, angka2)
    else:
        print(angka2, angka1)